enzyme/Enzyme/CApi.cpp

```diff
@@ -340,6 +340,21 @@ void EnzymeRegisterAllocationHandler(char *Name, CustomShadowAlloc AHandle,
     };
 }
 
+
+/// This is the main entry point to register a custom derivative for language frontends.
```

Comment:
I probably wouldn't put this here, can you put the docs [here](https://github.com/EnzymeAD/Enzyme/blob/31efe088a2c5d82f9f48323fee6e01fb3ecd5ef0/enzyme/Enzyme/GradientUtils.h#L87)  and cross reference/link it from here?